site/11ty/markdown.shortcut.js

```diff
@@ -1,11 +1,13 @@
-const path = require('path');
-const fsAsync = require('fs').promises;
+import path, {dirname} from 'path';
+import kleur from 'kleur';
```

Comment:
Cosmetic (same): it was more readable and useful (in case of migration) to use the alias `color` because it is exactly why we use this dependency.